### PR TITLE
eslint on src and src/__tests__

### PR DESCRIPTION
**Pull request by flovilmart**

### CODE REVIEW

File: src/Cloud.js [hunk 1]
```diff
@@ -59,23 +59,22 @@ export function run(
   return CoreManager.getCloudController().run(name, data, requestOptions);
 }
 
- /**
+/**
   * Gets data for the current set of cloud jobs.
   * @method getJobsData
   * @name Parse.Cloud.getJobsData
   * @param {Object} options
```
Comment (dplewis) [inline src/Cloud.js]: There are a few places where you removed options parameter but they still exist in the docs. 

https://github.com/gajus/eslint-plugin-jsdoc

or 

https://eslint.org/docs/rules/valid-jsdoc

Maybe we can use something like these plugin to validate the docs.

Comment (flovilmart) [reply src/Cloud.js]: Right, gotta have a look.

Comment (flovilmart) [reply src/Cloud.js]: valid-jsdoc yields an additional 200 errors ;) let's add it later ;)

Comment (dplewis) [reply src/Cloud.js]: I agree everything looks good to me here

Comment (flovilmart) [reply src/Cloud.js]: awesome!

**Pull request merged**